src/Tizen.Multimedia.Vision/MediaVision/InferenceModelConfiguration.cs

```diff
@@ -332,6 +340,30 @@ public InferenceTargetType Target
             }
         }
 
+        /// <summary>
+        /// Gets or sets the inference model's device.
+        /// </summary>
+        /// <remarks>
+        /// The default device is <see cref="InferenceTargetDevice.CPU"/>.<br/>
+        /// If a device doesn't support <see cref="InferenceTargetDevice.GPU"/> and <see cref="InferenceTargetDevice.Custom"/>,
+        /// <see cref="InferenceTargetDevice.CPU"/> will be used internally, despite the user's choice.
+        /// </remarks>
+        /// <exception cref="ArgumentException"><paramref name="value"/> is not valid.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public InferenceTargetDevice Device
+        {
+            get
+            {
+                return (InferenceTargetDevice)GetInt(_keyTargetType);
```

Comment:
Does `_keyTargetType` still be used? 
Actually `_keyTargetType` is "MV_INFERENCE_TARGET_TYPE".
so if you change `InferenceTargetType` to `InferenceTargetDevice`, I think you should add `_keyTargetDevice` and use it instead of `_keyTargetType`.
```
private const string _keyTargetDevice = "MV_INFERENCE_TARGET_DEVICE";
```

Reply:
Even though InferenceTargetDevice changes InferenceTargetType but they still specify cpu, gpu, or custom with changed enumeration values. 
So, InferenceTargetDevice is "MV_INFERENCE_TARGET_TYPE" and keep the _keyTargetType.

Reply:
If so, why did you change the name of it?
I thought they're pair.(`MV_INFERENCE_TARGET_TYPE` - `Inference TargetType`).